documents/openapi/openapi.yaml

```diff
@@ -1210,15 +1210,35 @@ paths:
               $ref: '#/components/schemas/PushNotificationRequest'
       responses:
         '201':
-          description: CREATED
-          content:
-            application/json:
-              schema:
-                type: object
-                properties:
-                  result:
-                    type: string
-                    example: "success"
+          $ref: '#/components/responses/PushCreated'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/ServiceAuthError'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+        '502':
+          $ref: '#/components/responses/BadGateway'
```

Comment:
The message for this response is `"Mobile app is not initialized"` - not sure how this would be a Bad Gateway, seems like 403 forbidden makes more sense here?

Reply:
I'm not sure about this either.  It already exists for the /v2/notifications/push route.

Reply:
Melanie said to just go with what exists now. Resolving comment.